pkgs/by-name/dc/dcm2niix/package.nix

```diff
@@ -63,6 +70,10 @@ stdenv.mkDerivation (finalAttrs: {
       "-DZLIB_IMPLEMENTATION=Cloudflare"
     ];
 
+  postInstall = lib.optionalString withPigz ''
+    wrapProgram $out/bin/dcm2niix --set PATH "${lib.makeBinPath [ pigz ]}"
```

Comment:
Would it make sense to rather prefix the PATH?

Reply:
Fine with me, changed.